includes/definitions/advafsp150eg-x.yaml

```diff
@@ -0,0 +1,13 @@
+os: advafsp150eg-x
+text: ADVA FSP150 EGX
+type: network
+icon: adva
+group: MetroE
```

Comment:
What's this? The group would be to tie in multiple OS' to one vendor, like cisco with ios, ios-xe, etc.